### PR TITLE
issue-37

### PR DESCRIPTION
**Pull request by mykolalosev**
### Description of your changes
Add the following resources:

- [x] [aws_docdb_cluster_parameter_group](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/docdb_cluster_parameter_group)
- [x] [aws_docdb_cluster_snapshot](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/docdb_cluster_snapshot)
- [x] [aws_docdb_event_subscription](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/docdb_event_subscription)

Fixes https://github.com/upbound/provider-aws/issues/37

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested
Manually and Uptest:
aws_docdb_cluster_parameter_group: https://github.com/upbound/provider-aws/actions/runs/3627832145
aws_docdb_cluster_snapshot: https://github.com/upbound/provider-aws/actions/runs/3627875937
aws_docdb_event_subscription: https://github.com/upbound/provider-aws/actions/runs/3627886219

### CODE REVIEW

**Comment by mykolalosev** [comment]
test-examples="examples/docdb/clusterparametergroup.yaml"

**Comment by mykolalosev** [comment]
test-examples="examples/docdb/clusterparametergroup.yaml"

**Comment by mykolalosev** [comment]
test-examples="examples/docdb/clusterparametergroup.yaml"

File: examples/docdb/eventsubscription.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      namespace: upbound-system
+    masterUsername: foo
+    preferredBackupWindow: 07:00-09:00
+    skipFinalSnapshot: true
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+    meta.upbound.io/example-id: docdb/v1beta1/clustersnapshot
```
Comment (ulucinar) [inline examples/docdb/eventsubscription.yaml:46]:
```suggestion
    meta.upbound.io/example-id: docdb/v1beta1/eventsubscription
```

**Review by ulucinar — DISMISSED** [review]
Thank you @mykolalosev.

**Review by ulucinar — COMMENTED** [review]
Are there any blockers for uptesting these resources in addition to the manual tests you performed? If there are, could you please add the manual intervention annotations?

**Comment by mykolalosev** [comment]
test-examples="examples/docdb/clusterparametergroup.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/docdb/clusterparametergroup.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/docdb/clustersnapshot.yaml"

**Comment by mykolalosev** [comment]
/test-examples="examples/docdb/eventsubscription.yaml"

**Review by ezgidemirel — APPROVED** [review]
Thanks a lot @mykolalosev!

**Pull request merged**